param/reactive.py

```diff
@@ -184,6 +191,38 @@ def len(self):
         rxi = self._reactive if isinstance(self._reactive, rx) else self()
         return rxi._apply_operator(len)
 
+    def map(self, func, *args, **kwargs):
```

Comment:
```suggestion
    def map(self, func, /, *args, **kwargs):
```

To match with https://docs.python.org/3/library/functools.html#functools.partial

Reply:
Thanks!